vyper/signatures/interface.py

```diff
@@ -85,18 +85,18 @@ def mk_full_signature_from_json(abi):
         for a in func['inputs']:
             arg = ast.arg(
                 arg=a['name'],
-                annotation=abi_type_to_ast(a['type']),
+                annotation=abi_type_to_ast(a['type'], 1048576),
```

Comment:
Add a comment that this supports a maximum-sized `string` or `bytes` dynamic structure of 1 Mb (1024 * 1024 * 1 byte)